sources/Renderer/Bootstrap/FieldRenderer/BsSelectObjectFieldRenderer.php

```diff
@@ -320,12 +320,19 @@ public function Render()
 			if ($this->oField->GetCurrentValue() !== null && $this->oField->GetCurrentValue() !== 0 && $this->oField->GetCurrentValue() !== '')
 			{
 				// Note : AllowAllData set to true here instead of checking scope's flag because we are displaying a value that has been set and validated
-				$oFieldValue = MetaModel::GetObject($sFieldValueClass, $this->oField->GetCurrentValue(), true, true);
+				$oFieldValue = MetaModel::GetObjectWithArchive($sFieldValueClass, $this->oField->GetCurrentValue(), true, true);
 				$sFieldHtmlValue = $oFieldValue->GetName();
-				$sFieldUrl = ApplicationContext::MakeObjectUrl($sFieldValueClass, $this->oField->GetCurrentValue());
-				if(!empty($sFieldUrl))
+				if($oFieldValue->IsArchived())
 				{
-					$sFieldHtmlValue = '<a href="'.$sFieldUrl.'" data-toggle="itop-portal-modal">'.$sFieldHtmlValue.'</a>';
+					$sFieldHtmlValue = '<span class="text_decoration"><span class="fas fa-archive object-archived fa-1x fa-fw"></span></span>' . $sFieldHtmlValue;
```

Comment:
```suggestion
					$sFieldHtmlValue = '<span class="text_decoration"><span class="fas fa-archive"></span></span>' . $sFieldHtmlValue;
```

I would defintively remove the `fa-1x fa-fw` classes as they are not necessary here, it will be more consistent with the other decorations (email, phone, ...).

Regarding the `object-archived` class, I have mixed feelings. We don't have convention yet for that type of meta-information in the portal, but we could still add it and migrate it in the future.

So once suggestion is applied, it will be ok for me.

Reply:
_I cant' apply the suggestion myself as the PR doesn't seem to allow editing for maintainers. The I'll push it through functional review on tuesday, then it should be merged._

Reply:
I always forget to set this option during PR creation and I am not able to change it later :-(

Anyhow I just committed your suggestions.

Thanks for your feedback and the review :-)

Reply:
Yes, as this option doesn't seem to exist anymore (or disabled for this repo?)
<img width="1257" alt="image" src="https://github.com/Combodo/iTop/assets/228588/89f047f3-50ae-4a05-851f-98b848474509">

Reply:
Hello,
I guess the option appearance would be based on the fork options instead of the initial repository options ? 

I'v just checked [the documentation](https://docs.github.com/en/github-ae@latest/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) again : there is now a red warning box stating "If your fork contains GitHub Actions workflows, the option is Allow edits and access to secrets by maintainers."

Also seems like it is again possible to change an existing PR : "Additionally, you can modify an existing pull request to let repository maintainers make commits to your branch." ?